liquibase-snowflake/src/test/java/liquibase/datatype/BinaryTypeSnowflakeTest.java

```diff
@@ -0,0 +1,73 @@
+package liquibase.datatype;
+
+import liquibase.database.core.OracleDatabase;
+import liquibase.database.core.SnowflakeDatabase;
+import liquibase.datatype.core.BinaryTypeSnowflake;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static liquibase.servicelocator.PrioritizedService.PRIORITY_DATABASE;
+import static org.junit.Assert.*;
+
+public class BinaryTypeSnowflakeTest {
+
+
+    BinaryTypeSnowflake binaryTypeSnowflake;
+    SnowflakeDatabase snowflakeDatabase;
+
+    @Before
+    public void setup() {
+        binaryTypeSnowflake = new BinaryTypeSnowflake();
+        snowflakeDatabase = new SnowflakeDatabase();
+    }
+
+    @Test
+    public void toDatabaseDataType() {
+        DatabaseDataType databaseDataType = binaryTypeSnowflake.toDatabaseDataType(snowflakeDatabase);
+        assertEquals("BINARY", databaseDataType.getType());
+        assertEquals("BINARY", databaseDataType.toSql());
+    }
+
+    @Test
+    public void binaryAliases() {
+        LiquibaseDataType liquibaseDataType = DataTypeFactory.getInstance().fromDescription("binary", snowflakeDatabase);
```

Comment:
Hello! The list of alias validation should include all of the aliases. Here is the list from the BinaryTypeSnowflake class:

"longblob"
"longvarbinary"
"java.sql.Types.BLOB"
 "java.sql.Types.LONGBLOB"
 "java.sql.Types.LONGVARBINARY"
 "java.sql.Types.VARBINARY"
 "java.sql.Types.BINARY"
"varbinary"
"binary"
"image"
"tinyblob"
"mediumblob

Reply:
working on it